src/components/sbb-checkbox-group/sbb-checkbox-group.scss

```diff
@@ -0,0 +1,46 @@
+// Default component properties, defined for :host. Properties which can not
```

Comment:
How can this situation be achieved?
<img width="412" alt="Bildschirmfoto 2022-11-15 um 17 02 38" src="https://user-images.githubusercontent.com/29278232/201967296-cfb1e8cc-407d-40f3-9eb1-72379433b09e.png">

Reply:
You can check it here: https://5ea7d9e48c451f00223b52ff-wvsrjfsaeo.chromatic.com/?path=/story/components-form-elements-sbb-checkbox-group--vertical-icon-end
Note that since it is done with "width: 100%", the checkbox is checkable by clicking in any point between the text and the icon.